src/Controls/samples/Controls.Sample/Pages/SemanticsPage.xaml

```diff
@@ -55,18 +55,18 @@
                 SemanticProperties.Hint="Hint text"/>
 
             <Entry
-                Text="Entry text TH"
+                Text="Entry text, iOS:DTH, Android:TH"
```

Comment:
I wonder how these read on XCT?

Like if we use the AccessibilityDelegate instead of the content description

Reply:
It reads the same on XCT at the moment

Reply:
will fix both here and in XCT